api/python/quilt3/packages.py

```diff
@@ -1395,30 +1404,42 @@ def dest_fn(lk, *args, **kwargs):
                         f"in the {registry!r} package registry specified by 'registry'."
                     )
 
+        if dedupe and not force:
+            raise QuiltException("dedupe=True requires force=True")
```

Comment:
Haven't thought much about it but why `dedupe` requires `force`?

Reply:
I'm also not sure that dedupe requires force.

Reply:
I'm not sure that force=True means "always push". I thought it means to skip only the specific check.
https://docs.quiltdata.com/api-reference/cli#push
```
  --force              Skip the parent top hash check and create a new
                       revision even if your local state is behind the remote
                       registry.
```

Reply:
You're right.

Reply:
That's how we use it, but "force" means "no matter what"... anway as long as we are consistent with how the app works.

Reply:
I only did it because dedupe without force seemed like a very obscure scenario: the parent hash has to match the latest hash and _also_ match the current package hash. Not sure how you end up in that scenario by accident (and if it's not by accident, then just don't call push unnecessarily?)

But anyways, I'll remove it.